spec/controllers/auctions_controller_spec.rb

```diff
@@ -9,6 +9,45 @@
       expect(auction).to be_a(AuctionViewModel)
       expect(auction.id).to eq(auction_record.id)
     end
+
+    context 'the list of auctions is sorted' do
+      it 'renders them descending by datetime' do
+
+        Timecop.freeze
+
+        date_start = Time.now - 3.days
```

Comment:
you want to use `Time.current` instead of `Time.now` -- might be reason for CI failures  http://winstonyw.com/2014/03/03/time_now_vs_time_zone_now/